### PR TITLE
Handle refresh_whatsapp_tokens task exceptions

### PR DESCRIPTION
**Pull request by koallann**
If some request error occurs in a WhatsApp channel during the `refresh_whatsapp_tokens` task, the remaining channels are not refreshed, so future messages to be sent will fail due to an invalid token.

### CODE REVIEW

File: temba/channels/types/whatsapp/tasks.py [hunk 1]
```diff
@@ -84,23 +84,26 @@ def refresh_whatsapp_tokens():
...
-                continue
+                if resp.status_code != 200:
+                    continue
 
-            channel.config["auth_token"] = resp.json()["users"][0]["token"]
-            channel.save(update_fields=["config"])
+                channel.config["auth_token"] = resp.json()["users"][0]["token"]
+                channel.save(update_fields=["config"])
+            except RequestException:
+                pass
```
Comment (norkans7) [inline temba/channels/types/whatsapp/tasks.py:106]: @koallann just log this exception to sentry here

**Review by norkans7 — CHANGES_REQUESTED** [review]
Looks good just log the exception

File: temba/channels/types/whatsapp/tasks.py (hunk 1 above)
Comment (koallann) [reply temba/channels/types/whatsapp/tasks.py]: @norkans7 done

**Review by norkans7 — APPROVED** [review]

**Review by norkans7 — CHANGES_REQUESTED** [review]
@koallann please add a unit test to cover that missed lines

**Comment by norkans7** [comment]
Added by https://github.com/nyaruka/rapidpro/pull/3022

**Pull request closed without merging**